geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/IngestCommand.scala

```diff
@@ -200,4 +200,10 @@ trait IngestParams extends OptionalTypeNameParam with OptionalFeatureSpecParam w
 
   @Parameter(names = Array("--src-list"), description = "Input files are text files with lists of files, one per line, to ingest.")
   var srcList: Boolean = false
+
+  @Parameter(names = Array("--no-job-status"), description = "Progress of the ingest job is not printed to the console, however, the application will close when job completes.")
+  var noJobStatusTracking: Boolean = false
```

Comment:
I understand why the options are written as "don't do ___", but the (inconsistent) double negatives throughout the code make it just that much more complicated to read.  Any chance we could rename these variables to be consistently `var jobStatusTracking: Boolean = true` and `var waitForCompletion: Boolean = true`?  Or, if that complicates the command-line annotation, create a second variable that is the positive assertion?  Or is it just me that's stuck on this?  (That happens.)

Reply:
I think we should keep the negative, as that allows you on the command line to do `--no-job-status` instead of `--job-status false`, which seems nicer IMO. But we could easily add a `def jobStatusTracking = !noJobStatusTracking` and use that in our code

Reply:
Although I think we should get rid of this particular parameter, but we could apply the theory to `--no-wait`